File: database/schemas/book.js
var database   = require('../bootstrap.js'); 
var Sequelize  = require('sequelize');

module.exports = database.define('Book', {
    isbn:            { type: Sequelize.STRING },
    title:           { type: Sequelize.STRING },
    author:          { type: Sequelize.STRING },
    publisher:       { type: Sequelize.STRING },
    publicationYear: { type: Sequelize.STRING },
});

File: src/library/book.py
class Book:

    def __init__(self, id=None, title=None):
        self.id = id
        self.title = title
        self.authors = []
        self.publisher = None
        self.publisher_place = None
        self.publication_year = None

    def __repr__(self):
        return '<Book id=%r>' % (self.id)
